### DISCUSSION TITLE
RPi2W Linux QZ - need BT disconnected from the RPI to turn off bike

### DISCUSSION THREAD

**Discussion by Vendo233** [cagnulein/qdomyos-zwift#2822] [General]
My bike Echelon GT is a simple bike which turns on with pedals move and turns off when disconnected from BT and pedals not moving. 

The Raspberry Pi is connected then to Samsung Tablet with Zwift /MyWhoosh. 

I will need to figure out how to SW or HW way turn off the QZ service

Option 1:
- Execute **on the tablet** SSH command for the Raspnerry Pi IP with "sudo systemctl start qz" or "sudo systemctl stop qz"
- something like this: https://play.google.com/store/apps/details?id=com.pd7l.sshbutton&hl=en_US&pli=1 

Option 2:
- Install physical switch for the RPI power, some sort of button and turn ON/OFF the whole device

any other option folks?

**Comment by cagnulein** [comment]
Can't you put qz on auto start and turn off the raspberry after inactivity?
Qz now has mqtt so you can auromatize it from home assistant or similar for
example

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno dom 24 nov 2024 alle 23:09 Vendo233 ***@***.***> ha
scritto:

**Comment by Vendo233** [comment]
Hello  Roberto, thank you very much for responding so quick, really exceptional!!!

The MQTT opens a lot of doors indeed, but how does one set the MQTT in the Linux ? 

I will search the github for clues , Im not up to date. Thank you!!

**Comment by cagnulein** [comment]
I will add the mqtt command line arguments if you need them
